core/services/feeds/config.go

```diff
@@ -17,4 +17,5 @@ type Config interface {
 	JobPipelineResultWriteQueueDepth() uint64
 	JobPipelineMaxSuccessfulRuns() uint64
 	MercuryCredentials(credName string) *ocr2models.MercuryCredentials
+	ThresholdKeyShare() string
```

Comment:
This interface is specific for Feeds. Maybe @jmank88 can help point to a place where a new interface should go. I imagine we could add a nice sub-structure similarly to Mercury.

Reply:
I tried to not put this in here and got a compilation error.  I think it might be required simply to conform to the interface.

Reply:
@cedric-cordenier is just beginning work to decouple these interfaces, but this is a particularly unusual one :thinking: 
Definitely open to renaming/refactoring whatever seems out of alignment.

Reply:
Somehow VRF secrets are not here and it works. core/config/secrets.go has an interesting and somewhat confusing struct as well..

Reply:
@bolekk VRF secrets do not pass config in this manner, however I think passing via the Delegate.cfg is the most straightforward way to pass the required key from `secrets.toml`.  @jmank88 As for decoupling these interfaces, should that work block this PR or can it be merged as-is?

Reply:
Maybe add a comment here saying "unused in Feeds, to be refactored" or something like that?

Reply:
No blocker. But if there is something small to improve along the way that would be fine.